src/script/campaign/mod/vco-listeners.lua

```diff
@@ -2,6 +2,27 @@ local vco = core:get_static_object("vco");
 
 -- COMMON --
 
+-- POOLED RESOURCES --
+
+local function update_faction_pooled_resource_earnings(faction_name, pooled_resource_key, amount)
+  if amount > 0 then
+	local current_amount_of_resource_earned = cm:get_saved_value("vco_pooled_resource_earned_" .. faction_name .. "_" .. pooled_resource_key) or 0;
+	cm:set_saved_value(
+	  "vco_pooled_resource_earned_" .. faction_name .. "_" .. pooled_resource_key,
+	  current_amount_of_resource_earned + amount
+	);
+  end
+
+  return false
+end
+
+local function has_faction_earned_gte_x_pooled_resource(faction, pooled_resource_key, target_amount)
+  local total_amount_of_resource_earned = cm:get_saved_value("vco_pooled_resource_earned_" .. faction .. "_" .. pooled_resource_key);
```

Comment:
Indentation :D

Reply:
Check you are using tabs instead of spaces. Is should be configurable in your IDE